syntax = "proto3";

import "google/protobuf/any.proto";

package grpc_template.v0;

service HelloService {
  rpc SayHello (HelloRequest) returns (HelloResponse);
}

message HelloRequest {
  string greeting = 1;
  google.protobuf.Any any = 2;
}

message HelloResponse {
  string reply = 1;
}